m4_sessions/1706-inning/group_four.md

```diff
@@ -1,63 +1,77 @@
+- making contributions in an industry where that's rewarded
 
-Feelings and thoughts:
 
-- people feel confident in their ability to contribute to the community
-- people have the knowledge of where to go to contribute
-- the knowledge of HOW to contribute (like OSS PR)
 
+# Session outline:
 
 
+### 1. Do we have a responsibility to "give back"?
 
-Ideas:
+### 2. Have attendees make list of all things they know AND all the ways they've been helped by folks in their current mod or mods ahead.  (put these questions on screen) (led by Valary)
 
-- videos
-- list of projects to contribute to
-- how to make an impact at Turing
-  - make PR on lessons
-  - make oss contributions
-  - PR checklist/template
+---- budgeting ten minutes to this point ----
 
-### Session Outline:
+### 3. watch [video](https://youtu.be/nXIIW8TusWw?t=9m58s) (8 min) (Sergey)
 
+---- now 18 mins in ----
 
-- do we have a responsibility to "give back" to the community?
+### 4. Brainstorm how they can get involved (turn and talk) (5 min) (Sergey)
 
-- what does "giving back" look like?
+### 5. collect their answers on white board (4 min), Discuss their answers (John B)
 
-High level, how helpful this can be
 
-- ever been helped by a blog?
-- ever been helped by another student in your cohort or a cohort ahead?
-- helped by a teacher?
-- a mentor?
+---- 27 min in ----
 
-We'll talk about how to do all that. 
 
-- OSS contributions (a guide?)
-- Write down skills you already have to build confidence
-- Find list of places in turing repo where you could find, fix, submit PR w/fix.
+### 6. Now we can share some of our ideas
+  - Making PR in Turing repo (John)
+  - host lunch-time in-mod session
 
-Walk through example of PR against Turing repo to fix problem
+  **To Discuss:**
 
-Walk through example of "student interest group", pairing, be a mentor
+> 1. Talk about how to make a PR against a lesson (gist w/instructions?) (John B) ([demo against this repo](https://github.com/turingschool/front-end-curriculum/blob/gh-pages/projects/ideabox.md))
+> 2. In-person involvement (like TryTuring). You're most qualified to help at a try-turing event after _mod 1_. (Kelley M)
+> 3. lunch & learn, spikes. Jump into channels to answer questions (Josh)
 
-Go outside the bubble, volunteer in the community (try turing, etc)
+--- ~ 45 mins in ---
 
-### Names of group members:
+### 7. Closing question: circle back to "do we have responsibility to give back?"
+  - who can giving back help the most?
+  - who do you want to make an impact on when you do give back? (consider yourself!)
 
+### 8. Give deliverables for PD session
+
+- Dates for upcoming Try Turing events
+- Valarie's PD template
+- Josh's PR practice (if you want)
+
+
+# Deliverables/resources
+
+They should join #metalearning (slack) to get all the details
+
+## Names of group members
+
+- Josh Thompson
 - Sergey L
 - Valery Trudell
 - Kelly Miller
 - John Binning
-- Josh Thompson
+
+
+
+we're "group 4" from:
```

Comment:
Great job with team!!  @josh-works Let me know if you have any questions on these comments.